### PR TITLE
Uses clu.metric_writer & adds config.log_every_step=100

### PR DESCRIPTION
**Pull request by andsteing**
- `clu.metric_writer` writes transparently both to TF summary files and to logging
- It's often useful to get training metrics before an epoch is fully finished; then it's also fine to report average values since the last emission of these metrics instead of logging the results of every batch (which also produces quite large TF summary files)

For comparison of metrics (eval metrics are identical, train metrics are much sparser):

- Before: https://tensorboard.dev/experiment/iJzNKovmS0q6k5t6k5wvOw/#scalars&_smoothingWeight=0&regexInput=v100_x8$
- This PR: https://tensorboard.dev/experiment/OTRYahNGS1m27exR3ryZvA/

### CODE REVIEW

**Review by jheek — APPROVED** [review]
LGTM! Sorry for the delay!

File: examples/imagenet/configs/default.py [hunk 1]
```diff
@@ -11,20 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# Copyright 2020 The Flax Authors.
```
Comment (jheek) [inline examples/imagenet/configs/default.py:15]: Was this removed accidentally?

Comment (andsteing) [reply examples/imagenet/configs/default.py:15]: The `# Copyright 2021` is enough, no?

Comment (andsteing) [reply examples/imagenet/configs/default.py:15]: (I just noted this passing by and thought I would include it in the same CL.)

Comment (jheek) [reply examples/imagenet/configs/default.py:15]: Oh I missed the duplicate. You are right

**Pull request merged**